prosemirror-jdita/src/nodes.ts

```diff
@@ -1,7 +1,18 @@
+/**
+ * TODO: Documentation
+ * // not used anywhere in the application currently
+ */
 export const NODES: Record<string, string> = {
   document: 'doc',
 };
 
+/**
+ * TODO: Documentation
```

Comment:
This needs to be resolved.

Reply:
This module isn't used, so I have updated the TODO as a "to-be-removed-later" comment. 
I suggest to remove this code in a follow-up PR. (d80de7b)

Reply:
This question has been moved to a list on the Petal google document, and the `TODO` has been removed from the code.